src/app/modules/verification/__tests__/verification.service.spec.ts

```diff
@@ -337,6 +348,45 @@ describe('Verification service', () => {
         expect(result._unsafeUnwrap()).toEqual(mockTransactionSuccessful)
       })
 
+      it('should send OTP with postman if admin has feature flag on', async () => {
```

Comment:
nit: if admin has feature flag `off`

Reply:
61918a066 updated the next testcase instead `should send OTP with twilio if admin has feature flag off` as admins would be turned on to use postman.